recipes/django-mptt-admin/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "django-mptt-admin" %}
+{% set version = "0.5.0" %}
+{% set sha256 = "a9fc3da9d819de6c3841fe3ff5f127c651ed2c97ac99150ab6adc728e08bdd86" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
Can you change this over to pip? https://conda-forge.org/docs/meta.html#use-pip

Reply:
Thanks for the review. Please let me know if you want more changes